index.html

```diff
@@ -10,6 +10,19 @@
     <link rel="stylesheet" href="styles/layout.css">
     <link rel="stylesheet" href="styles/modules.css">
     <link rel="icon" type="image/x-icon" href="images/peter-kim-profile.jpg">
+    <script id="project-template" type="text/x-handlebars-template">
```

Comment:
Good implementation of handlebars. Is there a reason you commented out the {{image}} handlebar?